externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/Constants.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.engine.spark
+
+object Constants {
```

Comment:
We should define `kyuubi.server.query.timeout` at server-side rather engine-side.

Reply:
Users can't infer time unit from `kyuubi.server.query.timeout`, and there are some configurations defined in second and others defined in millisecond, should we make the configuration name more clear?

Reply:
@pan3793 
I see the point, but the exists conf already named with `*.timeout`.. How about enhance the `kyuubi-defaults.conf.template` ? I guess it's useful if we write some common config into it.